.dockerignore

```diff
@@ -0,0 +1,6 @@
+venv/
```

Comment:
Hmm you could add a few more things like .circleci, docker-compose.yml, or you could also change the Dockerfile to only include what you need in the /app dir? Maybe that would be easier